web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v350/migrate-default.sql

```diff
@@ -11,4 +11,4 @@ UPDATE Settings SET value='3.5.0' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
 
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES
```

Comment:
I think this `INSERT` is not correct for a migration (not added in this PR anyway), to update the setting in a migration I think should be used some Java code, can you confirm this @juanluisrp

Reply:
I've created a PR to @MichelGabriel branch, https://github.com/MichelGabriel/core-geonetwork/pull/2.